package.json

```diff
@@ -3,8 +3,9 @@
   "private": true,
   "scripts": {
     "bootstrap": "lerna bootstrap --hoist",
-    "clean": "lerna run clean",
     "build": "lerna run build",
+    "clean": "lerna run clean",
+    "postinstall": "npm run bootstrap",
```

Comment:
On one hand I welcome this change because it's something probably most people will end up adding in their real life projects.

On the other hand, since this is a learning project, I feel that this is taking away information and could possibly cause confusion for someone trying to replicate this setup on their own. Bootstrapping is needed with lerna in order to make it work and in this example repo it's an explicit step to raise awareness around it.

I've updated the README a bit to explain when this step is needed. I hope that's a good compromise for now.

Reply:
I get your point. I personally think that running `lerna bootstrap` applies to pretty much any project using Lerna. I always try to reduce the setup to a minimum in order to make the on-boarding process as easy as possible.
I agree that this is a learning project and should be in sync with the content written on Medium.
I'm open to both ways. Feel free to close this pull request since this project is your baby :)

Reply:
After sleeping on this for a while I've decided to merge it in https://github.com/NiGhTTraX/lerna-ts/commit/c29d04d30691d485d06d5afbb4ce6517f2483ef2. Even though bringing awareness around bootstrapping has value, I realized that this repo and the associated blog post focus on the steps after a monorepo has been set up so removing any distracting setup is a plus. Thanks for creating this PR @marcobiedermann and making this repo a little bit better :)